Assets/MixedRealityToolkit/InputSystem/Sources/BaseGenericInputSource.cs

```diff
@@ -1,10 +1,11 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
-using Microsoft.MixedReality.Toolkit.Internal.Definitions.InputSystem;
+using Microsoft.MixedReality.Toolkit.Internal.Definitions;
 using Microsoft.MixedReality.Toolkit.Internal.Interfaces.InputSystem;
 using Microsoft.MixedReality.Toolkit.Internal.Managers;
 using System.Collections;
+using System.Collections.Generic;
```

Comment:
why include in some source files and not others that use Dictionary?

Reply:
Please elaborate @davidkline-ms ?

Reply:
Ok, sorted out the issue I believe you were pointing out.